recipe/meta.yaml

```diff
@@ -39,6 +39,7 @@ test:
     - test -e $PREFIX/lib/libmatio.dylib   # [osx]
     - if not exist %PREFIX%\Library\bin\libmatio.dll exit 1  # [win]
     - if not exist %PREFIX%\Library\lib\libmatio.lib exit 1  # [win]
+    - if not exist %PREFIX%\Library\include\matio.h exit 1  # [win]
```

Comment:
Shall we test as well against `matio_pubconf.h` ?

Reply:
The success of header copies is already checked in bld.bat.

Reply:
ok